docs/personalization/triggers.md

```diff
@@ -36,4 +37,8 @@ For each email trigger type, you need to decide on several crucial parameters, f
 - one or more [types of content](content_types.md)
 - [attributes](recommendation_models.md/#nominal-attributes) to be included in the response
 - time that must pass before email start being sent
-- number of repetitions
\ No newline at end of file
+- number of repetitions
+
+For the "Reactivation" model, you can decide what models and events are used to calculate the results.
+The following [recommendation models](recommendation_models.md) are used by default, in such order: "Also purchased", "Also clicked", and "Top purchased".
+If you do not decide otherwise, model calculations are based on BUY and TRANSFER [events](event_types.md).
```

Comment:
It is actually not based on BUY and TRANSFER, and model calculation is not done here. We just use `context event` to get model results. And in case of default models (and also null context) conext event is selected basing on model settings.